packages/mdc-icon-button/foundation.js

```diff
@@ -61,18 +61,10 @@ class MDCIconButtonToggleFoundation extends MDCFoundation {
 
```

Comment:
Remove `[de]registerInteractionHandler` from `get defaultAdapter()`, and from the adapter interface file